sample_env

```diff
@@ -18,7 +18,7 @@ GITLAB_TOKEN="<your GitLab token (unless you are working on code that interacts
 DEFINITION_STORE_PROVIDER="mongo"
 DEFINITION_MONGO_CONNECTION_STRING="mongodb://clearlydefined_mongo_db"
 DEFINITION_MONGO_DB_NAME="clearlydefined"
-DEFINITION_MONGO_COLLECTION_NAME="definitions-paged"
```

Comment:
Can take out this environment variable in the dev and production instance.  Can also add the new TRIMMED_DEFINITION_MONGO_COLLECTION_NAME variable to point "definitions-trimmed".  The default value for TRIMMED_DEFINITION_MONGO_COLLECTION_NAME is "definitions-trimmed"

Reply:
can we remove these now or after deployment?

Reply:
1. DEFINITION_MONGO_COLLECTION_NAME can be taken out now.  
2. DEFINITION_MONGO_TRIMMED_COLLECTION_NAME was renamed recently. The default value is "definitions-trimmed".  Can also add this one now as long as it points to "definitions-trimmed": 
- currently it does not have effect (due to the name change), 
- after service deployment, the value remains the same as the default that is currently used.